backend/src/app/services/snapshot/snapshot.service.ts

```diff
@@ -151,6 +152,121 @@ export class SnapshotsService {
     }
   }
 
+  getNotatioParticipantsForSnapshotCreation = async (notationId: string) => {
+    try {
+      if (notationId === '' || notationId === null) {
+        throw Error('notation id cannot be empty');
```

Comment:
Please implement HTTP exceptions with meaningful descriptive messages to enhance clarity and improve error handling.

Reply:
This is independent method  and no need for HTTP exception from here.